src/Admin/AdminInterface.php

```diff
@@ -225,6 +225,11 @@ public function getIdParameter();
      */
     public function hasRoute($name);
 
+    //    NEXT_MAJOR: declare native types in 4.0
```

Comment:
This comment is not needed, it's already planned to add typehint in next major ;)